dpdbaltics.php

```diff
@@ -159,6 +159,18 @@ public function hookActionFrontControllerSetMedia()
             $this->context->controller->addJS($this->getPathUri() . 'views/js/front/pudo.js');
             $this->context->controller->addJS($this->getPathUri() . 'views/js/front/pudo-search.js');
         }
+
+        if ($currentController === 'order-opc') {
+            $this->context->controller->addJS($this->getPathUri() . 'views/js/front/order-opc.js');
+            Media::addJsDef([
+               'order_opc_errors' => [
+                   'pickup_point_error' => $this->l('Please select pickup point'),
+                   'invalid_phone_error' => $this->l('Invalid phone number'),
+                   'empty_phone_error' => $this->l('Please fill up phone number'),
+                   'invalid_delivery_time' => $this->l('Please select delivery time')
+               ]
```

Comment:
in mollie project we faced issue when French people translating stuff by adding this symbol `'`. Try to add it, you will see that javascript will get broken . The solution to it is to use native php function addSlashes for texts, wrap each text with it and it should be fine - you can test this